sdk/android/azure-samples/src/main/java/com/azuresamples/MainActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.azuresamples;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.azure.data.appconfiguration.implementation.ClientConstants;
+import com.azuresamples.appconfiguration.HelloWorld;
+import com.azuresamples.appconfiguration.SecretReferenceConfigurationSettingSample;
+//import com.azure.data.appconfiguration.CreateSnapshot;
+import com.azuresamples.appconfiguration.WatchFeature;
+import com.azuresamples.appconfiguration.ConditionalRequestAsync;
+
+
+import com.azuresamples.storage.BasicExample;
+
+import java.io.IOException;
+
+public class MainActivity extends AppCompatActivity {
+
+
+    private final String[] keyvaultCredentials = {"Put keyvault credential string here here"};
+    private final String[] appconfigCredentials = {"Put appconfig credential string here"};
+    private final String[] storageCredentials = {"Put storage endpoint here", "Put storage secret key here"};
```

Comment:
No one will be manually running these tests - they need to work from the CI system.

Reply:
This should be sorted in latest updates